apps/crn-server/src/handlers/webhooks/webhook-squidex.ts

```diff
@@ -21,7 +21,16 @@ export const squidexWebhookFactory = (
   return lambda.http(squidexHandler);
 };
 
-const eventBridge = new EventBridge();
+const eventBridge = new EventBridge(
+  /* istanbul ignore next */
+  process.env.IS_OFFLINE
```

Comment:
put in the config maybe? just so we don't have process.env all over the place 

not a huge fan of such conditionals just for local dev tbh but meh

Reply:
I agree, I just wasn't sure how possible it was to emulate the pre-existing behaviour of sending literally nothing to `new EventBridge()` if defined in config, and I didn't want to risk breaking prod environments.

I guess I can try undefined parameters and see how it works though.

Reply:
Now updated, and seems to be working correctly in the review environment as far as I can tell.